SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#include "Async/Future.h"
+#include "CoreMinimal.h"
+#include "Modules/ModuleManager.h"
+#include "Serialization/JsonWriter.h"
+#include "Templates/SharedPointer.h"
+#include "Widgets/Notifications/SNotificationList.h"
+#include "FileCache.h"
```

Comment:
Predeclare as many of this as you can, in a bunch of other header files to.

Reply:
Good idea. Done